server/utils/utils.test.ts

```diff
@@ -510,3 +511,13 @@ describe('Get prisoner release date from Nomis', () => {
     })
   })
 })
+
+describe('Check if release date before cutoff date', () => {
+  it('should return true if release date is before cutoff date', () => {
+    expect(isReleaseDateBeforeCutOffDate('06/12/2023', '05/12/2023')).toBeTruthy()
+  })
+
+  it('should return false if release date is after cutoff date', () => {
+    expect(isReleaseDateBeforeCutOffDate('04/12/2023', '05/12/2023')).toBeFalsy()
```

Comment:
The tests should demo the behaviour and we're missing an important comparison - what if the dates are the same?